lib/util/detect_safari.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:html';
```

Comment:
Do you mind renaming this file to `detect_webkit`? I've been trying wherever possible to make sure no one thinks we're trying to throw Safari under the bus.

Reply:
yep, fixed.